calendars/skimage.yaml

```diff
@@ -6,11 +6,11 @@ events:
       The scikit-image Community Call. Friendly for Americas/Oceania.
 
       Meeting notes: https://hackmd.io/@alexdesiqueira/HyyHcIjQ9
-    begin: 2022-04-13 14:00:00 +00:00
-    end: 2022-04-13 15:00:00 +00:00
+    begin: 2022-04-13 21:00:00
```

Comment:
Do you want to change the times back? E.g., should this be:
```
    begin: 2022-04-13 14:00:00
```
instead of
```
    begin: 2022-04-13 21:00:00
```

Reply:
Good catch, thanks @jarrodmillman!